package.json

```diff
@@ -57,7 +57,7 @@
     "husky": "^7.0.4",
     "jest": "^27.5.1",
     "lint-staged": "^12.3.4",
-    "node-sass": "7.0.3",
+    "node-sass": "9.0.0",
```

Comment:
can we skip upgrading this in dev dependencies? i'd like to keep it on the lowest-supported version and should minimize this diff to one line (updating the peer deps)